website/sidebars.js

```diff
@@ -31,7 +31,6 @@ module.exports = {
         'submission/clinical-data-validation-rules',
         'submission/submitting-clinical-data',
         'submission/viewing-clinical-data',
-        'submission/molecular-data-prep',
```

Comment:
not sure how does this affect the menu... with the filename changes in the two .md files in this PR.
guess we'll find out soon enough